### PR TITLE
ovirt add wsgi to httpd build

### PR DESCRIPTION
**Pull request by mnecas**
add wsgi.patch for httpd
@mwperina

### CODE REVIEW

**Review by mwperina — COMMENTED** [review]
Is there a reason why not to add periodical removing artifacts to all wsgi based deployments? The configuration for removing artifacts is common for all deployments, so why it should be allowed only for apache wsgi deployments?

Juan/Paul, what do you think?

@jmolmo @pcuzner

**Review by jmolmo — APPROVED** [review]
I think it is sensible.

**Comment by mwperina** [comment]
> I think it is sensible.

Sorry Juan, I don't understand your reply. Do you think it would be better to have artifacts removal only for apache wsgi? Or do you think it should be allowed for all wsgi deployment?

@jmolmo

**Comment by jmolmo** [comment]
> > I think it is sensible.
> 
> Sorry Juan, I don't understand your reply. Do you think it would be better to have artifacts removal only for apache wsgi? Or do you think it should be allowed for all wsgi deployment?
> 
> @jmolmo

sorry ... too concies :-) .. I mean that is a good idea to have the artifacts removal for all the kinds of wsgi deployments. :-)

**Comment by mwperina** [comment]
> > > I think it is sensible.
> > 
> > 
> > Sorry Juan, I don't understand your reply. Do you think it would be better to have artifacts removal only for apache wsgi? Or do you think it should be allowed for all wsgi deployment?
> > @jmolmo
> 
> sorry ... too much concise :-) .. I mean that is a good idea to have the artifacts removal for all the kinds of wsgi deployments. :-)

Thanks for clarification! ;-)

File: misc/packaging/apache/wsgi.patch [hunk 1]
```diff
@@ -0,0 +1,24 @@
+diff --git a/wsgi.py b/wsgi.py
```
Comment (mwperina) [inline misc/packaging/apache/wsgi.patch:1]: Please make this change available for all wsgi deployments

**Review by mwperina — CHANGES_REQUESTED** [review]

File: misc/packaging/apache/wsgi.patch [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+index d66f9c5..86615de 100644
```
Comment (mwperina) [inline misc/packaging/apache/wsgi.patch:2]: What do we need this patch for? Isn't it part of common wsgi.py?

Comment (mnecas) [reply misc/packaging/apache/wsgi.patch:2]: For the apache and gunicorn the ansible_runner_service.py is used outside of the project.
When installing I moved it to the runner_service folder. But some parts of the project still use the original type.

Comment (mnecas) [reply misc/packaging/apache/wsgi.patch:2]: My bad accidentally kept the change when testing

**Review by mwperina — APPROVED** [review]
+1

**Comment by mnecas** [comment]
Done some tests and right after updating with this patch, it removed all artifacts older than 1 week. 
Can be merged.

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**